src/services/services.ts

```diff
@@ -375,6 +375,10 @@ namespace ts {
         constructor(_kind: SyntaxKind.Identifier, pos: number, end: number) {
             super(pos, end);
         }
+
+        get unescapedText(): string {
+            return unescapeLeadingUnderscores(this.text);

```

Comment:
Can we make this cacheing, since the node is supposed to be immutable? Ie, `return (delete this.unescapedText, this.unescapedText = unescapeLeadingUnderscores(this.text))`? (You have to use delete to remove the getter; otherwise you have to use an internal field to store the calculated value (and you still have a getter) - not really sure which is better, perf-wise - one has a branch on accesses, one doesn't)